landoapi/models/landing.py

```diff
@@ -63,14 +63,18 @@ def create(cls, revision_id, phabricator_api_key=None, diff_id=None):
         # save landing to make sure we've got the callback
         landing = cls(revision_id=revision_id, diff_id=diff_id).save()
 
-        trans = TransplantClient()
-        callback = '%s/landings/%s/update' % (
-            os.getenv('HOST_URL'), landing.id
+        # Define the pingback URL with the port
+        callback = '{host_url}:{pingback_port}/landings/{id}/update'.format(
```

Comment:
host_url here will be something different than the public facing url. Since this is taken from the env, no code needs to be changed, but noting here so you are all aware.